sdk/src/timing.rs

```diff
@@ -47,6 +47,12 @@ pub fn slot_duration_from_slots_per_year(slots_per_year: f64) -> Duration {
     Duration::from_nanos(slot_in_ns as u64)
 }
 
+/// From slots per year to slots per some minutes, rounded
+pub fn slots_per_interval(slots_per_year: f64, interval_mins: u64) -> u64 {
```

Comment:
this would be minutes_as_slots() ?  why "interval"?

Reply:
also: above, you mix ticks and slots line 41 "/// From slots per year to tick_duration", but the function is called `slot_duration_`

Reply:
> this would be minutes_as_slots() ? why "interval"?

No good reason. minutes_as_slots() is much better!